scripts/functions/checksum

```diff
@@ -198,14 +198,19 @@ __rvm_checksum_read()
     _checksum_md5="$(      __rvm_db_ "$rvm_path/config/md5"    "$_name" | \command \head -n 1 )"
     [[ -n "${_checksum_md5:-}" ]] ||
       _checksum_md5="$(    __rvm_db_ "$rvm_user_path/md5"      "$_name" | \command \head -n 1 )"
+    [[ -n "${_checksum_md5:-}" ]] ||
+      _checksum_md5="$(    __rvm_curl -s -L $_name.md5  | \command \grep -oe "[0-9a-f]{32}"  )"
 
     _checksum_sha512="$(   __rvm_db_ "$rvm_path/config/sha512" "$_name" | \command \head -n 1 )"
     [[ -n "${_checksum_sha512:-}" ]] ||
       _checksum_sha512="$( __rvm_db_ "$rvm_user_path/sha512"   "$_name" | \command \head -n 1 )"
+    [[ -n "${_checksum_md5:-}" ]] ||
+      _checksum_sha512="$( __rvm_curl -s -L $_name.sha512  | \command \grep -oe "[0-9a-f]{128}"  )"
```

Comment:
I wonder if we need a flag to enable/disable this as this will do extra queries to the web ... maybe we can at least check the `name` starts with `http`

Reply:
my bash isn't that good. would suggest you edit it for that

Reply:
Done!